libs/SalesforceHybrid/src/com/salesforce/androidsdk/phonegap/app/HybridApp.java

```diff
@@ -41,6 +41,14 @@ public void onCreate() {
 		super.onCreate();
 		SalesforceHybridSDKManager.initHybrid(getApplicationContext(), new HybridKeyImpl());
 
+		/*
+         * Uncomment the following lines to enable IDP login flow. This will allow the user to
+         * either authenticate using the current app or use a designated IDP app for login.
+         * Replace 'idpAppURIScheme' with the URI scheme of the IDP app meant to be used.
+         */
+		// SalesforceSDKManager.getInstance().setIDPLoginFlowEnabled(true);
```

Comment:
This is the code that needs to be uncommented to enable IDP login within the SP app.